### ISSUE TITLE
**Error in py_call_impl(callable, dots$args, dots$keywords) :    OSError: Unable to open file (file signature not found)**

### ISSUE THREAD

**Issue by dingjunxiang** [gao-lab/GLUE#45]
Hi
I am following evalution pipeline from the GLUE publication to reproduce the results ,and  meet a error.

**timeout 24h Rscript workflow/scripts/run_LIGER.R --input-rna results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/rna.h5ad --input-atac results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/frags2rna.h5ad -s 0 --output-rna results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/rna_latent.csv --output-atac results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/atac_latent.csv --run-info results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/run_info.yaml > results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/run_LIGER_FiG.log 2>&1 || touch results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/.blacklist**
Waiting at most 5 seconds for missing files.
MissingOutputException in line 120 of /root/GLUE/evaluation/workflow/rules/methods.smk:
Job Missing files after 5 seconds:
results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/rna_latent.csv
results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/atac_latent.csv
results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/run_info.yaml
This might be due to filesystem latency. If that is the case, consider to increase the wait time with --latency-wait.
Job id: 28634 completed successfully, but some output files are missing. 28634
Job failed, going on with independent jobs.
Select jobs to execute...

**Check the corresponding log file**(results/raw/Chen-2019/subsample_size:8000-subsample_seed:2/null/LIGER_FiG/default/seed:0/run_LIGER_FiG.log):

**Error in py_call_impl(callable, dots$args, dots$keywords) : 
  OSError: Unable to open file (file signature not found)**

Can someone please explain the error?

**Comment by Jeff1995** [comment]
Thanks for the question.

During our benchmarking, we did not obtain the ATAC fragment files for the `Chen-2019` dataset, so its "FiG" (Fragment in Genes) feature transformation could not be conducted. Accordingly, its `frags2rna.h5ad` files are just placeholders which help simplify the snakemake pipeline.

Integration methods running on these placeholder datasets would certainly throw errors, while at the same time create `.blacklist` files in their result directories. A second `snakemake` run will automatically exclude these failed runs and finish properly.

In short, these errors are expected. Just ignore them and run `snakemake` for a second time.

**Comment by dingjunxiang** [comment]
Thanks for your quick reply! The problem has been solved!

**Comment by Jeff1995** [comment]
Great!
